server/graphql/v2/object/Individual.js

```diff
@@ -75,6 +75,19 @@ export const Individual = new GraphQLObjectType({
           }
         },
       },
+      hasTwoFactorAuth: {
```

Comment:
I haven't looked at the updated Frontend yet but I'm wondering where it's needed. Just a note to remember checking that.

Reply:
Yep I have removed it from the larger query and created a small v2 query on Individual for the info needed.